contracts/staking/validatorShare/ValidatorShare.sol

```diff
@@ -22,71 +22,85 @@ contract ValidatorShare is ValidatorShareStorage {
 
     function updateCommissionRate(uint256 newCommissionRate) external onlyValidator {
         uint256 epoch = stakeManager.epoch();
+        uint256 _lastCommissionUpdate = lastCommissionUpdate;
+
         require( // withdrawalDelay == dynasty
-            (lastCommissionUpdate.add(stakeManager.withdrawalDelay()) <= epoch) || lastCommissionUpdate == 0, // For initial setting of commission rate
+            (_lastCommissionUpdate.add(stakeManager.withdrawalDelay()) <= epoch) || _lastCommissionUpdate == 0, // For initial setting of commission rate
             "Commission rate update cooldown period"
         );
+
         require(newCommissionRate <= 100, "Commission rate should be in range of 0-100");
         stakingLogger.logUpdateCommissionRate(validatorId, newCommissionRate, commissionRate);
         commissionRate = newCommissionRate;
         lastCommissionUpdate = epoch;
     }
 
-    function withdrawRewardsValidator() external onlyOwner returns (uint256 _rewards) {
-        _rewards = validatorRewards;
+    function withdrawRewardsValidator() external onlyOwner returns (uint256) {
+        uint256 _validatorRewards = validatorRewards;
```

Comment:
both are the same things right?

Reply:
Right, just trying to keep code consistent where function returns variable instead of assigning to return variable.